src/components/Navigation/index.js

```diff
@@ -1012,24 +1015,25 @@ class Navigation extends Component {
           </div>
           )}
         <ReactMapGL
-          latitude={viewport.latitude}
-          longitude={viewport.longitude}
-          zoom={viewport.zoom}
-          bearing={viewport.bearing}
-          pitch={viewport.pitch}
+          initialViewState={{
+            longitude: viewport.longitude,
+            latitude: viewport.latitude,
+            zoom: viewport.zoom,
+          }}
+          onLoad={e => this.geolocateControlRef.current && this.geolocateControlRef.current.trigger()}
```

Comment:
Could we avoid requesting the user's location unless they click on the map? That would preserve the current behavior

Reply:
Yes, we can change it to onClick.